Makefile

```diff
@@ -11,13 +11,50 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 CMDS=nfsplugin
-all: build
+IMAGE_NAME ?= nfs-csi
+REGISTRY ?= andyzhangx
+IMAGE_TAG_LATEST = $(REGISTRY)/$(IMAGE_NAME):latest
+DEPLOY_FOLDER = ./deploy/kubernetes
+.PHONY: all
+all: nfs
 
 include release-tools/build.make
 
 .PHONY: sanity-test
-sanity-test: build
+sanity-test: nfs
 	./test/sanity/run-test.sh
 
+.PHONY: verify
+verify:
+	hack/verify-all.sh
+
+.PHONY: nfs
+nfs:
+	CGO_ENABLED=0 GOOS=linux go build -a -o _output/nfsplugin ./cmd/nfsplugin
+
+.PHONY: container
+container: nfs
+	docker build --no-cache -t $(IMAGE_TAG_LATEST) -f ./cmd/nfsplugin/dev.Dockerfile .
```

Comment:
current error is
```
docker build --no-cache -t andyzhangx/quay.io/k8scsi/:latest -f ./cmd/nfsplugin/dev.Dockerfile .
invalid argument "andyzhangx/quay.io/k8scsi/:latest" for "-t, --tag" flag: invalid reference format
```

caused by `IMAGE_NAME` overridden by `release-tools/build.make`

should be `docker build --no-cache -t andyzhangx/nfs-csi:latest -f ./Dockerfile .`